cpp/src/arrow/gpu/cuda_context.h

```diff
@@ -37,13 +37,33 @@ class ARROW_EXPORT CudaDeviceManager {
   static Status GetInstance(CudaDeviceManager** manager);
 
   /// \brief Get the shared CUDA driver context for a particular device
+  /// \param[in] device_number
+  /// \param[out] out shared context
   Status GetContext(int gpu_number, std::shared_ptr<CudaContext>* ctx);
 
+  /// \brief Get the shared CUDA driver context for a particular device
```

Comment:
The use of "shared" here and in the `\brief` for `GetContext` might confuse. Maybe this should be "cached"?

Reply:
Yes, using "cached" in `GetContext` `\brief` is more appropiate.
Fixed.